### PR TITLE
Tseganesh.blogs

### PR DESCRIPTION
**Pull request by TseganeshYifruBerta**
Fix naming convention

### CODE REVIEW

**Comment by TseganeshYifruBerta** [comment]
image

**Comment by TseganeshYifruBerta** [comment]
localhost_3000_blog_blogs (7)

File: Web/starter_project_vanguard/components/blog/Blog.tsx [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+                children={'April 20'}
+                color="black"
+                weight="light"
+              ></Text>
+            </div>
+          </div>
+
+          <div className="flex flex-wrap">
+            <div>
+              {' '}
```
Comment (jebessaD) [inline Web/starter_project_vanguard/components/blog/Blog.tsx:148]: Remove an unnecessary code `{' '}`

File: Web/starter_project_vanguard/components/blog/Blog.tsx [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+              <Title />{' '}
+            </div>
+            <div>
+              {' '}
```
Comment (jebessaD) [inline Web/starter_project_vanguard/components/blog/Blog.tsx:152]: Remove an unnecessary code `{' '}`

File: Web/starter_project_vanguard/components/blog/Blog.tsx [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+  return (
+    <div className="flex flex-wrap justify-start items-end">
+      {tags.map((tag, index) => (
+        <Chip key={index} label={tag} />
+      ))}
+    </div>
+  )
+}
+
+function Description() {
```
Comment (jebessaD) [inline Web/starter_project_vanguard/components/blog/Blog.tsx:21]: Use JSON data for description, tags, avater ....

File: Web/starter_project_vanguard/components/blog/Blog.tsx [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+    <div className="flex items-center px-4 py-2 rounded-full bg-gray-200 text-gray-400 mr-2 mb-2 mx-4 mt-5 ">
+      <div className="text-md font-normal">{label}</div>
+    </div>
+  )
+}
+
+function Title() {
+  return (
+    <div>
+      {/* <Text size="2xl" family="montserrat" children={'Yididya Kebede'} color="black"></Text> */}
```
Comment (jebessaD) [inline Web/starter_project_vanguard/components/blog/Blog.tsx:104]: Do not leave commented code in the code, that will be merged in the main.

**Review by jebessaD — COMMENTED** [review]
I left some comments on your code.

**Pull request closed without merging**